CONTRIBUTORS.txt

```diff
@@ -189,6 +189,7 @@ contributors:
   Added new check 'comparison-with-callable'.
   Removed six package dependency.
   Added new check 'chained-comparison'.
+  Added new check 'useless-object-inheritance'.
```

Comment:
At this point you can stop adding these :) Just add `many more` 🥇

Reply:
will do it from the next commit :)